### DISCUSSION TITLE
Scroll page shortcuts?

### DISCUSSION THREAD

**Discussion by gzzinx** [ddterm/gnome-shell-extension-ddterm#254] [Q&A]
Is there way to bind scrolling multiple lines up/down, e.g. on Ctrl+Shift+Up/Down?

**Comment by amezin** [comment]
~~No, these shortcuts aren't implemented yet.~~

You can use Shift+PageUp/PageDown.
However, they aren't configurable. Seems to be a built-in Gtk feature.
